numpy/_core/src/multiarray/scalartypes.c.src

```diff
@@ -2035,29 +2036,33 @@ gentype_getarray(PyObject *scalar, PyObject *args)
     return ret;
 }
 
-static char doc_sc_wraparray[] = "sc.__array_wrap__(obj) return scalar from array";
+static char doc_sc_wraparray[] = "Array wrap to implementation for scalar types";
```

Comment:
delete "to" - and maybe still spell out the dunder method for easier grepping, i.e., `"__array_wrap__ implementation for scalar types"`